test/Microsoft.ML.Tests/Transformers/NormalizerTests.cs

```diff
@@ -207,6 +208,118 @@ public void NormalizerParameters()
             Done();
         }
 
+        [Fact]
+        public void NormalizerParametersMultiColumnApi()
+        {
+            string dataPath = GetDataPath("iris.txt");
+
+            var loader = new TextLoader(Env, new TextLoader.Options
+            {
+                Columns = new[] {
+                    new TextLoader.Column("float1", DataKind.Single, 1),
+                    new TextLoader.Column("float4", DataKind.Single, new[]{new TextLoader.Range(1, 4) }),
+                    new TextLoader.Column("double1", DataKind.Double, 1),
+                    new TextLoader.Column("double4", DataKind.Double, new[]{new TextLoader.Range(1, 4) }),
+                    new TextLoader.Column("int1", DataKind.Int32, 0),
+                    new TextLoader.Column("float0", DataKind.Single, new[]{ new TextLoader.Range { Min = 1, VariableEnd = true } })
+                },
+                HasHeader = true
+            }, new MultiFileSource(dataPath));
+            var context = new MLContext(seed: 0);
+            var est = context.Transforms.NormalizeMinMax(
+                new[] { new InputOutputColumnPair("float1"), new InputOutputColumnPair("float4"),
+                    new InputOutputColumnPair("double1"), new InputOutputColumnPair("double4"), }).Append(
+                    context.Transforms.NormalizeBinning(
+                        new[] {new InputOutputColumnPair("float1bin", "float1"), new InputOutputColumnPair("float4bin", "float4"),
+                            new InputOutputColumnPair("double1bin", "double1"), new InputOutputColumnPair("double4bin", "double4")})).Append(
+                    context.Transforms.NormalizeMeanVariance(
+                        new[] {new InputOutputColumnPair("float1mv", "float1"), new InputOutputColumnPair("float4mv", "float4"),
+                            new InputOutputColumnPair("double1mv", "double1"), new InputOutputColumnPair("double4mv", "double4")})).Append(
+                    context.Transforms.NormalizeLogMeanVariance(
+                        new[] {new InputOutputColumnPair("float1lmv", "float1"), new InputOutputColumnPair("float4lmv", "float4"),
+                            new InputOutputColumnPair("double1lmv", "double1"), new InputOutputColumnPair("double4lmv", "double4")}));
```

Comment:
How about `NormalizeSupervisedBinning`? Do we have a test for that?

Reply:
Just added within this test.

---
In reply to: [271496017](https://github.com/dotnet/machinelearning/pull/3172#discussion_r271496017) [](ancestors = 271496017)